app/src/test/java/ch/epfl/sdp/blindly/DateTest.kt

```diff
@@ -12,17 +13,17 @@ class DateTest {
     @Test
     fun getAgeComputeTheCorrectAge() {
         val TEST_AGE = 20
-        val TEST_DATE : DateInterface = Date(5, 1, 2001)
+        val TEST_DATE = Date(5, 1, 2001)
         assertThat(TEST_DATE.getAge(), equalTo(TEST_AGE))
     }
 
     @Test
     fun getDateReturnCorrectDateWhenBirthdayIsNotNull() {
         val TEST_BIRTHDAY = "05.01.2001"
-        val TEST_DATE = Date(5,1,2001)
+        val TEST_DATE = Date(5, 1, 2001)
         val date = Date.getDate(TEST_BIRTHDAY)
         if (date != null) {
-            assertThat(date == TEST_DATE, equalTo(true))
+            assertThat(date == TEST_DATE, `is`(true))
```

Comment:
Shouldn't it be like that?
```suggestion
            assertThat(date, equalTo(TEST_DATE))
```

Reply:
I was lazy, so I didn't want to write another test to test the equal.... xD
But yes it wouuld be way better